CryptomatorCloudAccess.xcodeproj/project.pbxproj

```diff
@@ -872,7 +928,6 @@
 				4A058FED24519FFC008831F9 /* Sources */,
 				4A058FEE24519FFC008831F9 /* Frameworks */,
 				4A058FEF24519FFC008831F9 /* Resources */,
-				74862A722469A7AF003D81CB /* Lint With SwiftFormat */,
```

Comment:
Oh, no there is no reason! I actually wanted to remove this temporarily, because it showed me too many warnings, which I wanted to fix once at the end.
Resolved by 75e00433a62e473a297b3bb2802de3e39238bcab.